### PR TITLE
docs: added plugin version issue notice to contributing guide [APE-869]

### PR DESCRIPTION
**Pull request by Ninjagod1251**
### What I did

Add the contributing notice

fixes: #90 
Fixes: APE-114

### Checklist

- [ ] All changes are completed
- [ ] New test cases have been added
- [ ] Documentation has been updated

### CODE REVIEW

**Review by fubuloubu — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ source venv/bin/activate
 pip install -e .'[dev]'
 ```
 
+##### NOTE: You might run into issues where you have a local install and are trying to work with a plugin pinned to a specific version.
```
Comment (antazoey) [inline CONTRIBUTING.md:18]: this is the wrong number of `#`

Comment (antazoey) [inline CONTRIBUTING.md:18]: `#` is for titles, not notes.
For notes do this use the asterisks. (Look at other parts of the docs with the bolded notes part.)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,6 +15,10 @@ source venv/bin/activate
...
+
+[The easy solution](https://github.com/ApeWorX/ape/issues/90) to this is simple fetch the tags via `git fetch upstream --tags` and reinstall via `pip install ..` You will then have the correct version.
```
Comment (antazoey) [inline CONTRIBUTING.md:20]: 1 sentence per line

Comment (antazoey) [inline CONTRIBUTING.md:20]:
```suggestion
[The easiest solution](https://github.com/ApeWorX/ape/issues/90) to this is to fetch the tags via `git fetch upstream --tags` and reinstall via `pip install ..`.
You will then have the correct version.
```

**Review by antazoey — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -15,6 +15,11 @@ source venv/bin/activate
 pip install -e .'[dev]'
 ```
 
+**NOTE**: You might run into issues where you have a local install and are trying to work with a plugin pinned to a specific version.
+
+[The easiest solution](https://github.com/ApeWorX/ape/issues/90) to this is to fetch the tags via `git fetch upstream --tags` and reinstall via `pip install ..`
```
Comment (antazoey) [inline CONTRIBUTING.md:20]:
```suggestion
[The easiest solution](https://github.com/ApeWorX/ape/issues/90) to this is to fetch the tags via `git fetch upstream --tags` and reinstall via `pip install .`. 
```

**Review by antazoey — APPROVED** [review]

**Pull request merged**